experimental/adaptive-dialog/docs/anatomy-and-runtime-behavior.md

```diff
@@ -0,0 +1,82 @@
+# Adaptive dialog: Anatomy and runtime behavior
+
+## Anatomy: Adaptive dialog
+
+***Adaptive dialogs*** at the core comprise of 4 main concepts - 
+- [Recognizers](#Recognizers)
+- [Rules](#Rules)
+- [Steps](#Steps)
+- [Inputs](#Inputs)
+
+<p align="center">
+    <img alt="Adaptive_dialog_anatomy" src="./Assets/adaptive-dialog-anatomy.png" style="max-width:700px;" />
+</p>
+
+### Recognizers
+_Recognizers_ help understand and extract meaningful pieces of information from user's input. All recognizers emit events - of specific interest is the 'recognizedIntent' event that fires when the recognizer picks up an intent (or extracts entities) from given user utterance. See [here][1] to learn more about supported recognizers and their usage.
+
+### Rules
```

Comment:
Reading this make we think maybe we should just name rules events and call it a day. It will simplified the entire need to explain that rules are event. Later we can address steps within an event as the 'handler' etc. Should we consider that post Build?